src/util/psbt/map/input.rs

```diff
@@ -12,18 +12,19 @@
 // If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
 //
 
-use std::collections::BTreeMap;
+use std::collections::{btree_map::Entry, BTreeMap};
 
 use blockdata::script::Script;
 use blockdata::transaction::{SigHashType, Transaction, TxOut};
 use consensus::encode;
+use hashes::{self, hash160, ripemd160, sha256, sha256d};
 use util::bip32::{DerivationPath, Fingerprint};
 use util::key::PublicKey;
 use util::psbt;
 use util::psbt::map::Map;
 use util::psbt::raw;
-use util::psbt::Error;
-
+use util::psbt::serialize::Deserialize;
+use util::psbt::{Error, error::PsbtHash};
```

Comment:
It's not clear that `PsbtHash` is a error type: we either need to have it as `error::PsbtHash`, or rename it into something indicating error...